packages/form-control/src/FormControlMixin.ts

```diff
@@ -143,6 +149,15 @@ export function FormControlMixin<
      * @private
      */
     #onInvalid = (): void => {
+      if (this.#awaitingValidationTarget && this.validationTarget) {
+        this.internals.setValidity(
+          this.validity,
+          this.validationMessage,
+          this.validationTarget
+        );
```

Comment:
how can we set the validity on the validationTarget if we're awaiting it? i guess its ok if the validationTarget is null here?

Reply:
The idea here is that earlier we've noted that the validationTarget did not exist, but line 152 here checks to see if it has been created since before.

We don't have to do extra work, having the element focus on itself is fine if the host is focusable. This saves all of the setInterval work from before (hence the deleted test).